mk_verificator/utils/__init__.py

```diff
@@ -38,7 +38,18 @@ def get_groups(config):
 
 
 def get_configuration(path_to_test):
+    """global conf is minor, but local conf is major."""
+
```

Comment:
then let's reorganize a bit our logic:
1) load global_config
2) check if local config for test
3)
if local_config:
   global_config.update(local_config)

In my vision it makes sense